profile/README.md

```diff
@@ -0,0 +1,138 @@
+<div align="center">
+    <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/E-constellatio.svg" alt="Logo" height="150px">
+    <h1>
+        Welcome in E-constellatio Github organization
+    <img src="https://media.giphy.com/media/hvRJCLFzcasrR4ia7z/giphy.gif" alt="Hi hand" width="30px"/>
+    </h1>
+</div>
+
+This organization contain repositories of E-constellatio website. Repositories of other products can be found
```

Comment:
Since the website for E-constellatio is planned to not hold advanced logics, shouldn't we only have one repository ? 
```suggestion
This organization contains repositories of E-constellatio website. Repositories of other products can be found
```